packages/relay/src/services/PostService.ts

```diff
@@ -251,11 +251,15 @@ export class PostService {
         return txHash
     }
 
-    async fetchSinglePost(id: string, db: DB): Promise<Post | null> {
+    async fetchSinglePost(
+        id: string,
+        status: number,
+        db: DB
+    ): Promise<Post | null> {
         const post = await db.findOne('Post', {
```

Comment:
we can modify where clause as below to make error message more clear.
```typescript
const whereClause = {
    postId: id
}

if (status) {
    whereClause[status] = status
}
```

Reply:
What part of the error might be unclear?